scripts_src/newreno/nceldrid.ssl

```diff
@@ -320,6 +324,17 @@ procedure pickup_p_proc begin
    end
 end
 
+procedure ensure_pistol begin
```

Comment:
I don't think this is necessary.

Reply:
Too much code for an ephemeral issue. Just let it stay wiped.

Reply:
No.